test/lib/ansible_test/_data/completion/docker.txt

```diff
@@ -1,13 +1,13 @@
-default name=quay.io/ansible/default-test-container:2.7.0 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined context=collection
-default name=quay.io/ansible/ansible-base-test-container:1.6.0 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined context=ansible-base
+default name=quay.io/ansible/default-test-container:2.9.0 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined context=collection
+default name=quay.io/ansible/ansible-base-test-container:1.7.0 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined context=ansible-base
 alpine3 name=quay.io/ansible/alpine3-test-container:1.19.0 python=3.8
 centos6 name=quay.io/ansible/centos6-test-container:1.17.0 python=2.6 seccomp=unconfined
 centos7 name=quay.io/ansible/centos7-test-container:1.17.0 python=2.7 seccomp=unconfined
-centos8 name=quay.io/ansible/centos8-test-container:1.17.0 python=3.6 seccomp=unconfined
+centos8 name=quay.io/ansible/centos8-test-container:1.20.0 python=3.6 seccomp=unconfined
```

Comment:
We might as well make this and the other version 20 containers use 21 since that's the latest tag.